packages/@okta/vuepress-site/docs/guides/quickstart/create-org/cli/create-org.md

```diff
@@ -0,0 +1,21 @@
+Creating an org is easy using the [Okta CLI](https://github.com/oktadeveloper/okta-cli):
+
+1. After installing the Okta CLI, run the command `okta register`.
+
+2. You are prompted to supply the following information:
+
+	 - First name
+	 - Last name
+	 - Email address
+	 - Company
+
+3. Okta creates the org. The CLI returns your org URL, also known as your Okta domain, and lets you know that a verification code is being sent to your email.
+
+	Make a note of your org URL: It’s the base URL you use to access your org.  Authorization requests for users will be directed to an endpoint that has this as its base, and any Okta API endpoints you call will also have this URL as their base.
```

Comment:
If possible, it'd help for us to be very intentional about how we use `org URL` and `Okta domain`. They are not always equivalent.

An `Okta domain` (`dev-000000.okta.com`, `id.example.com`, etc) is not a URL. It does not include a protocol or scheme. We should say and mean `Okta domain` in most circumstances.

`Org URL` has ambiguity about whether `-admin` is included, and it would include a protocol or scheme. We should avoid saying `org URL` unless describing something (UI label, SDK config key) that that is still using that term.

Reply:
@alexdahl-okta, since this step is guiding the reader through using the CLI, I had to use the term the CLI uses, which is org URL.

Reply:
@alexdahl-okta Do we need to change the words used by the Okta CLI? If so, please enter an issue. https://github.com/oktadeveloper/okta-cli

Reply:
@motikagan-okta Yeah, if a tool like the CLI says `org URL`, we should say `org URL` when referring to what the tool tells you.

But otherwise we should avoid conflating the two.

Here:

- `The CLI returns your org URL, also known as your Okta domain` is false. They are not the same thing.
- `Make a note of your org URL` is perpetuating a bad pattern: we should say `Okta domain`.

I think the treatment on https://developer.okta.com/docs/guides/sign-into-web-app/aspnet/configure-packages/ is great: `Okta Domain — Find it on the Developer Console dashboard in the upper-right corner as the Org URL.`
- Clearly establishes that the canonical term for this value is `Okta domain`.
- But tells you what it appears as in a legacy UI, so that you can still find it.